Philips.CodeAnalysis.Test/Maintainability/Maintainability/LockObjectsMustBeReadonlyAnalyzerTest.cs

```diff
@@ -62,13 +64,15 @@ public void Test()
 	}}
 }}
 ";
-			var result = Array.Empty<DiagnosticResult>();
 			if (isError)
 			{
-				result = new[] { DiagnosticResultHelper.Create(DiagnosticId.LocksShouldBeReadonly) };
+				var result = new[] { DiagnosticResultHelper.Create(DiagnosticId.LocksShouldBeReadonly) };
+				VerifyDiagnostic(string.Format(template, field), result);
 			}
-
-			VerifyDiagnostic(string.Format(template, field), result);
+            else
```

Comment:
indentation seems off

Reply:
I will be making another pass to simplify all the unnecessary [] building in the test classes.